pkg/scheduler/algorithm/priorities/image_locality_test.go

```diff
@@ -131,7 +158,7 @@ func TestImageLocalityPriority(t *testing.T) {
 			// Pod: gcr.io/40 gcr.io/140
 
 			// Node1
-			// Image: gcr.io/40 40MB, gcr.io/140 140MB
+			// Image: gcr.io/40:default_tag 40MB, gcr.io/140:default_tag 140MB
```

Comment:
Could you please change `default_tag` to `latest`? It's better to read in comment. :-)